src/speed-dial/index.tsx

```diff
@@ -33,11 +66,46 @@ export const Action = actionFactory(({ properties, children, middleware: { theme
 			}}
 			classes={classes}
 			variant={variant}
+			aria={
+				label
+					? {
+							'aria-describedby': labelId
```

Comment:
the `aria` property shouldn't be prefixed by `aria` as this is done automatically. You can see the result in the browser `aria-aria-describedby`.
<img width="386" alt="Screenshot 2021-03-30 at 14 49 51" src="https://user-images.githubusercontent.com/814453/112999833-30201500-9167-11eb-85e6-8538db5fbcb3.png">.
Could you add a specific `Action` test to capture this please? Currently it's untested.